assets/js/components/KeyMetrics/MetricTileNumeric.js

```diff
@@ -43,19 +44,20 @@ export default function MetricTileNumeric( props ) {
 		subText,
 		previousValue,
 		currentValue,
+		error,
 	} = props;
 
 	const formatOptions = expandNumFmtOptions( metricValueFormat );
 
-	return (
+	return ! error ? (
```

Comment:
Returning early in the case of an error would be more aligned with the general codebase, e.g.

```js
if ( error ) {
	return (
		<MetricTileError error={ error } headerText={ title } />
	);
}

return (
	<Widget noPadding>
		// ...
);